### ISSUE TITLE
CKPT_model

### ISSUE THREAD

**Issue by ArthurConan** [huanzhang12/ZOO-Attack#4]
Hello again, I really enjoy your program
But can you help me with such question: this tool works with ImageNet model ".pb"
But is it possible to import "model.ckpt" ?
Thanks in advance

**Comment by huanzhang12** [comment]
Yes it is possible. You can convert a "model.ckpt" to a ".pb". You can follow the "How to evaluate my own model" section in this tutorial on my other repository:
https://github.com/huanzhang12/CLEVER

**Comment by ArthurConan** [comment]
o, many thanks
I'll follow it now

**Comment by ArthurConan** [comment]
sorry for bothering you
i want to test by your ZOO attack the model "ens_adv_inception_resnet_v2_2017_08_18.tar.gz"

but i tried to use your tutorial and failed
and tried to convert by tensorflow export_inference_graph.py file and also failed
cause i get such error

File "test_all.py", line 262, in <module>
    main(args)
  File "test_all.py", line 102, in main
    data, model = ImageNet(), InceptionModel(sess, use_log)
  File "/root/Documents/python/ZOO-Attack-master/setup_inception.py", line 225, in __init__
    self.model = InceptionModelPrediction(sess, use_log)
  File "/root/Documents/python/ZOO-Attack-master/setup_inception.py", line 192, in __init__
    return_elements=[output_name])
  File "/usr/local/lib/python3.6/dist-packages/tensorflow/python/util/deprecation.py", line 432, in new_func
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/tensorflow/python/framework/importer.py", line 753, in import_graph_def
    % ', '.join(absent_input_keys))
ValueError: Attempted to map inputs that were not found in graph_def: [input:0]

**Comment by huanzhang12** [comment]
If you are able to generate the frozen probobuf, you are very close to your goal :)

You need to change the input name in line 173, 221 and 227 in `setup_inception.py` and replace the "input:0" to the input tensor name that you observed in TensorBoard for you model. You also need to do the same for all the output nodes (like "InceptionV3/Predictions/Softmax:0" and "InceptionV3/Predictions/Reshape:0"). Replace them with the name of output nodes of your new graph. The current names are for inception v3 only.
